### ISSUE TITLE
tf_sentencpiece requires GLIBC_2.27 

### ISSUE THREAD

**Issue by jsnowacki** [google/sentencepiece#364]
The current version of `tf_sentencepiece` installed from PyPI, i.e.:
```
pip install sentencepiece  tf-sentencepiece
```
works only on Linux which has GLIBC_2.27 (or potentially newer, but I didn't try). For older system like Debiam 9 stretch you get error as follows:
```
NotFoundError                             Traceback (most recent call last)
<ipython-input-3-2ebcca794b82> in <module>
      2 import tensorflow_hub as hub
      3 import numpy as np
----> 4 import tf_sentencepiece

~/.local/lib/python3.5/site-packages/tf_sentencepiece/__init__.py in <module>
      3 from __future__ import print_function
      4 
----> 5 from tf_sentencepiece.sentencepiece_processor_ops import *

~/.local/lib/python3.5/site-packages/tf_sentencepiece/sentencepiece_processor_ops.py in <module>
     41 
     42 
---> 43 _gen_sentencepiece_processor_op = tf.load_op_library(so_file)
     44 
     45 

/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/load_library.py in load_op_library(library_filename)
     59     RuntimeError: when unable to load the library or get the python wrappers.
     60   """
---> 61   lib_handle = py_tf.TF_LoadLibrary(library_filename)
     62 
     63   op_list_str = py_tf.TF_GetOpList(lib_handle)

NotFoundError: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by /home/jupyter/.local/lib/python3.5/site-packages/tf_sentencepiece/_sentencepiece_processor_ops.so.1.14.0)
```
Debian 9 is quite widely used. In our case we use it on GCP AI Platform, as Debian 9 is the official Deep Learning image for the notebooks. 

The simplest workaround that we test now is using Ubuntu 18.04 bionic, which works fine with `tf_sentencepiece` and TF 1.14.0 at the moment. This can be tested e.g. in Colab notebook https://colab.research.google.com/github/tensorflow/hub/blob/master/examples/colab/cross_lingual_similarity_with_tf_hub_multilingual_universal_encoder.ipynb, which works currently on Ubuntu 18.04; I've manually changed TF version to 1.14.0, as notebook has originally 1.12.0.

**Comment by kamal4493** [comment]
I have the same problem using google deep learning VM's

**Comment by DanzaMacabra** [comment]
I ran into the same error running notebook in official tensorflow 1.13.2 docker.
libc.so.6 linked to libc-2.23.so in the said docker, I have to upgrade manually to 2.27 since highest version candidate returned by "apt-cache policy libc" is libc-2.23.

**Comment by TTrapper** [comment]
Can confirm I am also having this issue on Ubuntu 16.04 with `tensorflow 1.14.0` and `tf-sentencepiece 0.1.82.1`

**Comment by ramizdundar** [comment]
I ran this problem too and my versions are same as @TTrapper

**Comment by chiragjn** [comment]
I was able to compile from source and build the wheel using the script here https://github.com/google/sentencepiece/blob/master/tensorflow/make_py_wheel.sh,

Had to change the `D_GLIBCXX_USE_CXX11_ABI` flag to `1` on line 47 as mentioned in https://github.com/google/sentencepiece/issues/293#issuecomment-510806920

However, sentencepiece still refused to register the ops in tf registry, so I had to manually load the correct .so file using `tf.load_op_library`

**Comment by taku910** [comment]
Recent;y, tensorflow team has released an official docker image to build custom-op. I've rebuild tf_sentencepiece with this docker image and released it as v0.1.83. You can install the latest one via "pip install tf_sentencepiece".

https://github.com/google/sentencepiece/releases/tag/v0.1.83
